packages/core/src/domain/report/reportObservable.ts

```diff
@@ -0,0 +1,119 @@
+import { mergeObservables, Observable } from '../../tools/observable'
+import { DOM_EVENT, includes, addEventListener } from '../../tools/utils'
+import { monitor } from '../internalMonitoring'
+import type {
+  Report,
+  BrowserWindow,
+  ReportType,
+  ReportingObserverCallback,
+  ReportingObserverOption,
+  ReportingObserver as ReportingObserverInterface,
+} from './browser.types'
+
+declare const ReportingObserver: {
+  new (callback: ReportingObserverCallback, option: ReportingObserverOption): ReportingObserverInterface
+}
+
+export const CustomReportType = {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what do you want to express with "custom"?

Reply:
I just don't want to conflict with native types that lives in `browser.types.ts`.

Reply:
I can also use an alias for the native types that I import. Wdyt?

Reply:
To avoid conflict, we usually prefix those names with `Rum`: `RumReportType`. WDYT?

Reply:
I thought that we use `RUM` and `Logs` prefixes when the code is specific to one package like:
`Configuration`, `RumConfiguration`, `LogsConfiguration`

I think `RumReportType` will be misleading. Wdyt?

Reply:
Agreed, we use `Raw` prefix at several places, could it fit here?

Reply:
Ah right, it is used in Logs too. `Raw` seems fine then. In my mind, `Custom` indicates that it is somehow customer related.

Reply:
Okay, I'm good with Raw ;)